### PR TITLE
Auto-update Glitch with GitHub

### LINKED ISSUE #95: Auto-update Glitch with GitHub
Resources: https://support.glitch.com/t/tutorial-how-to-auto-update-your-project-with-github/8124

### PR DESCRIPTION
**Pull request by AmanRaj1608**
Fixes #95 

Auto updating Glitch using glitch webhook
I have used this blog for fixing issue - https://dev.to/healeycodes/continuous-deployment-to-glitch-via-github-webhooks-3hmh

### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Non Functional Requirement
- [ ] Follows the code style of this project.
- [ ] Tests Cover Changes
- [ ] I have performed a self-review of my own code
- [ ] All new and existing tests passed.
- [ ] Documentation

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -3,6 +3,9 @@ const express = require('express')
 const app = express()
 const axios = require('axios')
 const path = require('path')
+const crypto = require('crypto')
+const execSync = require('child_process')
+const bodyParser = require('body-parser')
```
Comment (AmanRaj1608) [inline server.js]: Do I need to run `npm install body-parser --save` or it's already available

File: server.js [hunk 2]
```diff
@@ -18,6 +21,30 @@ const b18 = process.env.B18
...
+app.post('/git', (req, res) => {
+  const hmac = crypto.createHmac('sha1', glitch)
+  const sig = 'sha1=' + hmac.update(JSON.stringify(req.body)).digest('hex')
+  if (req.headers['x-github-event'] === 'push' &&
+    crypto.timingSafeEqual(Buffer.from(sig), Buffer.from(req.headers['x-hub-signature']))) {
+    res.sendStatus(200)
+    const commands = [
+      'git fetch origin master',
+      'git reset --hard origin/master',
+      'git pull origin master --force',
```
Comment (AmanRaj1608) [inline server.js]: is there any other command need to be added

**Comment by aashutoshrathi** [comment]
image

Hook is ready!

File: server.js (hunk 2 above)
Comment (aashutoshrathi) [reply server.js]: Yes, also add `npm i`.

File: server.js [hunk 3]
```diff
@@ -18,6 +21,30 @@ const b18 = process.env.B18
...
+      'refresh']
+    for (const cmd of commands) {
+      console.log(execSync(cmd).toString())
```
Comment (yashLadha) [inline server.js]: What if any of the command failed or some exception has occured?

Comment (aashutoshrathi) [reply server.js]: Yeah!
We might need validations after each command is executed.

**Comment by AmanRaj1608** [comment]
> Can you just test this one locally? before we begin the review.

I tried this same code on one of my own project and it works fine.

**Comment by AmanRaj1608** [comment]
@aashutoshrathi @yashLadha 
Please review again

**Pull request merged**